currency_rate_update/views/res_config_settings.xml

```diff
@@ -13,9 +13,15 @@
             <field name="module_currency_rate_live" position="attributes">
                 <attribute name="invisible">1</attribute>
             </field>
+            <label for="module_currency_rate_live" position="attributes">
```

Comment:
You shouldn't do in this module. If you don't want that, use `res_config_enterprise_remove` (not sure the exact name) module.

Reply:
I updated the view and left both options...